checks/check_extra730

```diff
@@ -27,24 +27,34 @@ CHECK_DOC_extra730='https://docs.aws.amazon.com/config/latest/developerguide/acm
 CHECK_CAF_EPIC_extra730='Data Protection'
 
 extra730(){
+  # Only RSA key types, needed to recover Amazon Issued, Imported and Private PKIs
+  local ACM_KEY_TYPES="RSA_1024,RSA_2048,RSA_3072,RSA_4096"
+  local ACM_CERTIFICATE_STATUSES="ISSUED"
+
   # "Check if ACM Certificates are about to expire in $DAYS_TO_EXPIRE_THRESHOLD days or less"
   for regx in $REGIONS; do
-    LIST_OF_ACM_CERTS=$($AWSCLI acm list-certificates $PROFILE_OPT --region $regx --query 'CertificateSummaryList[].CertificateArn' --output text)
-    if [[ $LIST_OF_ACM_CERTS ]];then
+    LIST_OF_ACM_CERTS=$("${AWSCLI}" acm list-certificates ${PROFILE_OPT} --region "${regx}" --include keyTypes="${ACM_KEY_TYPES}" --certificate-statuses "${ACM_CERTIFICATE_STATUSES}" --query 'CertificateSummaryList[].CertificateArn' --output text)
```

Comment:
What about imported certificates?

Reply:
It checks every ACM certificate whether it is Amazon Issued, Imported or comes from a Private PKIs.

For that reason we have included two filters:
- `ACM_KEY_TYPES`: to set the algorithms that can be used to generate key pairs.
- `ACM_CERTIFICATE_STATUSES`: to only check ACM Certificates whose status is `ISSUED`. The following statuses are not checked:
```
PENDING_VALIDATION
INACTIVE
EXPIRED
VALIDATION_TIMED_OUT
REVOKED
FAILED
```

Reply:
It checks every ACM certificate whether it is Amazon Issued, Imported or comes from a Private PKI.

For that reason we have included two filters:
- `ACM_KEY_TYPES`: to set the algorithms that can be used to generate key pairs.
- `ACM_CERTIFICATE_STATUSES`: to only check ACM Certificates whose status is `ISSUED`. The following statuses are not checked:
```
PENDING_VALIDATION
INACTIVE
EXPIRED
VALIDATION_TIMED_OUT
REVOKED
FAILED
```